source/fundamentals/builders/filters.txt

```diff
@@ -11,3 +11,571 @@ Filters Builders
+
+   * - :java-core-api:`lt() <com/mongodb/client/model/Filters.html#lt(java.lang.String,TItem)>`
+     - values less than a specified value.
+
+   * - :java-core-api:`lte() <com/mongodb/client/model/Filters.html#lte(java.lang.String,TItem)>`
+     - values less than or equal to a specified value.
+
+   * - :java-core-api:`ne() <com/mongodb/client/model/Filters.html#ne(java.lang.String,TItem)>`
+     - values not equal to a specified value.
+
+   * - :java-core-api:`in() <com/mongodb/client/model/Filters.html#in(java.lang.String,java.lang.Iterable)>`
+     - any of the values specified in an array.
+
+   * - :java-core-api:`nin() <com/mongodb/client/model/Filters.html#nin(java.lang.String,java.lang.Iterable)>`
+     - none of the values specified in an array.
+
+   * - :java-core-api:`empty() <com/mongodb/client/model/Filters.html#empty()>`
+     - all the documents.
+
+The following example creates a filter that matches all documents where
+the value of the ``qty`` field equals "5" in the ``paint_purchases`` collection:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/Filters.java
+   :language: java
+   :dedent:
+   :start-after: begin equalComparison
+   :end-before: end equalComparison
+   
+The JSON output renders as the following:
+
+.. code-block:: json
+   :copyable: false
+   
+   { "$eq": { "qty": 5 } }
+
+The output of the above code snippet should look something like this:
+
+.. code-block:: json
+   :copyable: false
+   
+   {"_id": 1, "color": "red", "qty": 5, "vendor": ["A"]}
+   {"_id": 6, "color": "pink", "qty": 5, "vendor": ["C"]}
+
+The following example creates a filter that matches all documents where
+the value of the ``qty`` field is greater than or equal to "8" in the
+``paint_purchases`` collection: 
+
+.. literalinclude:: /includes/fundamentals/code-snippets/Filters.java
+   :language: java
+   :dedent:
+   :start-after: begin gteComparison
+   :end-before: end gteComparison
+   
+The JSON output renders as the following:
+
+.. code-block:: json
+   :copyable: false
+   
+   { "$gte": { "qty": 8 } }
+
+The output of the above code snippet should look something like this:
+
+.. code-block:: json
+   :copyable: false
+   
+   {"_id": 2, "color": "purple", "qty": 10, "vendor": ["C", "D"]}
+   {"_id": 3, "color": "blue", "qty": 8, "vendor": ["B", "A"]}
+   {"_id": 5, "color": "yellow", "qty": 11, "vendor": ["A", "B"]}
+   {"_id": 7, "color": "green", "qty": 8, "vendor": ["B", "C"]}
+
+The following example creates a filter that matches all documents in
+the ``paint_purchases`` collection because the predicate is empty: 
+
+.. literalinclude:: /includes/fundamentals/code-snippets/Filters.java
+   :language: java
+   :dedent:
+   :start-after: begin emptyComparison
+   :end-before: end emptyComparison
+
+The JSON output renders as the following:
+
+.. code-block:: json
+   :copyable: false
+   
+   { }
+
+The output of the above code snippet should look something like this:
```

Comment:
I think the user would get the idea if you mentioned here "The output of the code snippet above consists of all the documents in the collection." and then showed just the first three followed by an ellipsis. That way, we don't need to show all 8 lines to try to explain it.